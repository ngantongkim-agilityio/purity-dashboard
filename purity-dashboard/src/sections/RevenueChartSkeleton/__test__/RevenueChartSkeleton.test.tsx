import { render } from '@testing-library/react';

// Components
import { RevenueChartSkeleton } from '..';

describe('RevenueChartSkeleton component', () => {
  it('Should render RevenueChartSkeleton component with default props', () => {
    const { container } = render(<RevenueChartSkeleton />);

    expect(container).toMatchSnapshot();
  });
});
