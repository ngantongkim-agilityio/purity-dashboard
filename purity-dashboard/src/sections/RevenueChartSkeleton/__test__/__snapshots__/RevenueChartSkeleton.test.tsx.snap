// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RevenueChartSkeleton component Should render RevenueChartSkeleton component with default props 1`] = `
<div>
  <div
    class="before:absolute before:inset-0 before:-translate-x-full before:animate-[shimmer_2s_infinite] before:bg-gradient-to-r before:from-transparent before:via-white/60 before:to-transparent relative w-full overflow-hidden md:col-span-4"
  >
    <div
      class="mb-4 h-8 w-36 rounded-md bg-gray-100"
    />
    <div
      class="rounded-xl bg-gray-100 p-4"
    >
      <div
        class="sm:grid-cols-13 mt-0 grid h-[410px] grid-cols-12 items-end gap-2 rounded-md bg-white p-4 md:gap-4"
      />
      <div
        class="flex items-center pb-2 pt-6"
      >
        <div
          class="h-5 w-5 rounded-full bg-gray-200"
        />
        <div
          class="ml-2 h-4 w-20 rounded-md bg-gray-200"
        />
      </div>
    </div>
  </div>
</div>
`;
